### ISSUE TITLE
Anyway to specify the config to build with?

### ISSUE THREAD

**Issue by FaultyFunctions** [nommiin/builder#13]
![image](https://user-images.githubusercontent.com/6508045/103467641-b8348100-4d16-11eb-9c71-ad7a72a76c25.png)
This thing ^^

**Comment by nommiin** [comment]
Yup, you should be able to! When builder is loaded, it should create a "Configs" folder in your resource tree. You can double click a config and it should change the project name at the top!

![image](https://user-images.githubusercontent.com/6483989/103467732-8bcd3480-4d17-11eb-8288-4972154dc589.png)

Let me know if it isn't working for some reason :>

**Comment by FaultyFunctions** [comment]
Hmm this isn't working for me for some reason. I'm on the latest V1.2 of builder and I checked with the newest source of GMEdit as well as the binary released on 12-19-2019. The Configs Folder doesn't show up for me.

![image](https://user-images.githubusercontent.com/6508045/103467822-a8b63780-4d18-11eb-8c98-c9c386c09e4d.png)

Is there a setting I'm missing or should it just work? (I can confirm that builder works as far as building and running the game goes).

EDIT: I'm on GMS2.3 latest w/ runtime 409.

**Comment by nommiin** [comment]
Hmm... very odd, opening 2.3 projects seems to add the configs folder just fine for me.

Is there any sort of error thrown in the Chrome dev console upon loading the project? Also if possible, could you upload your configuration files so I could compare them with mine? You can find them at `%appdata%\AceGM\GMEdit\config`

**Comment by FaultyFunctions** [comment]
Yeah so it seems that these lines are "causing" the error:
![image](https://user-images.githubusercontent.com/6508045/103468173-20399600-4d1c-11eb-92f9-cd38b348684d.png)
If I comment it out, the configs folder loads fine albeit with just the default config because those lines are probably parsing the configs I have set in the project.

Here is the Chrome dev console error with those lines kept in:
![image](https://user-images.githubusercontent.com/6508045/103468208-54ad5200-4d1c-11eb-82e0-8695fafca736.png)
I thought it might be the TransFX extension causing it but when I removed it to test, it just picked up another file as causing the error.

Here are my config files in a ZIP: [config_files.zip](https://github.com/nommiin/builder/files/5760977/config_files.zip)

**Comment by FaultyFunctions** [comment]
So digging a bit deeper I decided to log what was actually going into the JSON.parse function. I used this to log it to the console:
```js
console.log(new TextDecoder("utf-8").decode(Electron_FS.readFileSync(Project.current.path)));
```
Here is what I got:
```json
{
  "resources": [
    {"id":{"name":"TransFX","path":"extensions/TransFX/TransFX.yy",},"order":0,},
    {"id":{"name":"fnt_debug","path":"fonts/fnt_debug/fnt_debug.yy",},"order":0,},
    {"id":{"name":"fnt_debug_large","path":"fonts/fnt_debug_large/fnt_debug_large.yy",},"order":1,},
    {"id":{"name":"fnt_shell","path":"fonts/fnt_shell/fnt_shell.yy",},"order":2,},
    {"id":{"name":"Animation","path":"objects/Animation/Animation.yy",},"order":0,},
    // Rest of the entries
    ]
}
```
It seems like this isn't proper JSON? It doesn't like the comma's after the path and order keys. It should be this apparently:
```json
{
  "resources": [
    {"id":{"name":"TransFX","path":"extensions/TransFX/TransFX.yy"},"order":0},
    {"id":{"name":"fnt_debug","path":"fonts/fnt_debug/fnt_debug.yy"},"order":0},
    {"id":{"name":"fnt_debug_large","path":"fonts/fnt_debug_large/fnt_debug_large.yy"},"order":1},
    {"id":{"name":"fnt_shell","path":"fonts/fnt_shell/fnt_shell.yy"},"order":2},
    {"id":{"name":"Animation","path":"objects/Animation/Animation.yy"},"order":0}
    // Rest of the entries
    ]
}
```

I'm not really sure what's going on here...

**Comment by brady-goldsworthy** [comment]
This might help someone...

I was getting the same issue as @FaultyFunctions when installing the plugin normally, using the downloads from the release.

The issue was resolved when I directly cloned from master into the GMEdit Plugins folder. Not sure why that would resolve it, or if it's just a fluke.
